utils/clients/blizzard/client.js

```diff
@@ -1,6 +1,4 @@
-import { revalidate } from '../../../app/page';
-import getAccessToken from '../../db/getAccessToken';
-import updateAccessToken from '../../db/updateAccessToken';
+import { cacheAccessToken, cacheUpdateAccessToken } from '../../redis/client';
```

Comment:
You might have already noticed, but it looks like there's a circular dependency between the blizzard & redis clients. That's generally considered to be a bad thing but it's not the end of the world. This is probably something that should be refactored in the future.